ivy/functional/frontends/numpy/statistics/correlating.py

```diff
@@ -0,0 +1,8 @@
+import ivy
+
+
+def sum(x, /, *, axis=None, dtype=None, keepdims=False, out=None):
```

Comment:
According to the document of numpy.sum, here you need two extra optional keyward argument called initial and where. Also, the initial value of keepdims should be None.
Here is the original arguments of numpy.sum: numpy.sum(a, axis=None, dtype=None, out=None, keepdims=<no value>, initial=<no value>, where=<no value>)

Reply:
Well, the frontend function we create should have the exact same behaviour as the origenal function in the targeted framework.

Reply:
the backend sum function losses **initial** and **where**